### PR TITLE
LPS-132503 Only try to reload the iframe if have a valid contentWindow

### PR DESCRIPTION
**Pull request by boton**
**Description:** https://issues.liferay.com/browse/LPS-132503

If the iframe isn't loaded because of a submit or navigation the contentWindow is null.

https://html.spec.whatwg.org/multipage/iframe-embed-object.html#dom-iframe-contentwindow
> The contentWindow IDL attribute must return the WindowProxy object of the iframe element's nested browsing context, if its nested browsing context is non-null, or null otherwise.

In this case, trying to access to iframe.contentWindow.location throws an error.

**Note:** Don´t forward directly forward to FI.

### CODE REVIEW

**Comment by boton** [comment]
ci:test:sf

**Comment by boton** [comment]
ci:test:relevant

File: modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/progress.js [hunk 1]
```diff
@@ -81,7 +81,11 @@ AUI.add(
 					var instance = this;
 
 					setTimeout(() => {
-						instance._frame.get('contentWindow.location').reload();
+						if (instance._frame.get('contentWindow')) {
```
Comment (carloslancha) [inline modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/progress.js:84]: Could we use just `instance._frame.get('contentWindow.location')?.reload();`?

Comment (boton) [reply modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/progress.js:84]: Nop, the method `instance._frame.get` fails inside. It's doesn't expect `null` only `undefined`.
Maybe we can use something like `instance._frame._node.contentWindow?. location.reload()` but doesn't look good.

Comment (boton) [reply modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/progress.js:84]: Do you prefer to check the node directly `instance._frame._node.contentWindow?.location.reload()`? 
It's a little uglier for me.

Do have you another idea or suggestion?

Comment (carloslancha) [reply modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/progress.js:84]: If we can do `instance._frame.get('contentWindow')?.location.reload()` I'd go for it, if not just keep your changes ;)

Comment (boton) [reply modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/progress.js:84]: I'm going to try it right now :D

Comment (boton) [reply modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/progress.js:84]: With a `?`  more works like a charm. Thanks!

`instance._frame.get('contentWindow')?.location?.reload()`

**Comment by boton** [comment]
See https://github.com/liferay-frontend/liferay-portal/pull/1132.

**Pull request closed without merging**